File: java/factorial.java
public class factorial {
    
    public static void main(String[] args) {
        System.out.println(" find the factorial of ");
        int n=8 ,fact=1;

        for (int i = 1; i <=n; i++)
        {
            fact = fact*i;
        }
        
        System.out.println("factorial of n is " +fact);
    }
}

File: RUBY/wd201/lev3/recursion/factorial.rb
puts "Enter a number to find factorial"
num = gets.chomp.to_i
fact = 1
if num == 0
  puts "error"
else
  i = 1
  while (i <= num)
    p fact = fact * i
    i += 1
  end
end
puts "factorial of #{num} is #{fact}"

